server/clients/manageRecallsApi/manageRecallsApiClient.ts

```diff
@@ -57,6 +57,14 @@ export function addRecallDocument(
   return restClient(token).post({ path: `/recalls/${recallId}/documents`, data: document })
 }
 
+/*
+Temporarily send the userId in the request body until it is added to the JWT token and can be retrieved by the API
+ */
+export function addUserDetails(userId: string, firstName: string, lastName: string, token: string): Promise<Recall> {
+  const request = { userId, firstName, lastName }
+  return restClient(token).post<Recall>({ path: '/users', data: request })
```

Comment:
The general type you're sending through is a Recall, should be a new type for User?

Reply:
doh!  Cut and paste shenanigans